rolling_update.yml

```diff
@@ -59,7 +63,7 @@
       until: result.rc == 0
       retries: 5
       delay: 10
-      delegate_to: 127.0.0.1
+      delegate_to: "{{ groups.mons[0] }}"
```

Comment:
localhost is not working for me, since ansible mgt machine is not a part of the ceph cluster.

Oh, I see your concern. How about this?

```
- name: select a running monitor
  set_fact: mon_host={{ item }}
  with_items: groups.mons
  when: item != inventory_hostname

- name: Waiting for clean PGs...
  ...
  delegate_to: {{ mon_host }}
```